### PR TITLE
Link Bitrix24 icon. RVK & Skiline Icons do not work. Why?

### PR DESCRIPTION
**Pull request by peterge1998**
## Description

Fixes #(issue)

## Type of change

:x: Bug fix (non-breaking change which fixes an issue)
:x: Icon addition (non-breaking change that adds/modifies Lawnicons's icons)
:x: General change (non-breaking change that doesn't fit the above categories like copyediting)

### Icons added
* App Name (`package.name`) - added from icon request form
* App Name (`package.name`)
* App Name (`package.name`)

### Icons updated
* App Name (`package.name`)
* App Name (`package.name`)

### Icons linked
* App Name (linked `package.name` to `@drawable/package`)
* App Name (linked `package.name` to `@drawable/package`)

### CODE REVIEW

File: app/assets/appfilter.xml [hunk 1]
```diff
@@ -36,11 +36,14 @@
...
   <item component="ComponentInfo{com.android.calendar/com.android.calendar.homepage.AllInOneActivity}" drawable="themed_icon_calendar_31" name="Calendar"/>
   <!--  Lawnicons -->
+  <item component="ComponentInfo{com.bitrix24.android/com.bitrix24.android.BX24Activity}" drawable="bitrix24" name="Bitrix24"/>
   <item component="ComponentInfo{de.pietsmiet.watchbuddy/com.example.pietsmiet_live_event_app.MainActivity}" drawable="watchbuddy" name="watchbuddy"/>
   <item component="ComponentInfo{com.tunnelbear.android/com.tunnelbear.android.mvvmReDesign.NavGraphActivity}" drawable="tunnelbear" name="Tunnelbear"/>
   <item component="ComponentInfo{tipster.evolutioit.app/info.androidhive.evolutioit.ui.common.base_activity.BaseActivity}" drawable="tipster" name="Tipster"/>
   <item component="ComponentInfo{se.virtualtrainer.dumbbell/se.starkapps.starkfitness.MainActivity}" drawable="stark_hantel" name="Stark Dumbell"/>
+# The Skiline Icon does not work
   <item component="ComponentInfo{cc.skiline.android/cc.skiline.android.screens.main.MainActivity}" drawable="skiline" name="Skiline"/>
```
Comment (daywalk3r666) [inline app/assets/appfilter.xml:45]: <item component="ComponentInfo{cc.skiline.android/cc.skiline.android.screens.startup.StartupActivity}"

File: app/assets/appfilter.xml [hunk 2]
```diff
@@ -36,11 +36,14 @@
...
+# The RVK Icon does not work
   <item component="ComponentInfo{koeln.kvb.ticket.rvkapp/com.cubic.cumo.mts.MainActivity}" drawable="rvk" name="RVK"/>
```
Comment (daywalk3r666) [inline app/assets/appfilter.xml:47]: <item component="ComponentInfo{koeln.kvb.ticket.rvkapp/com.cubic.cumo.mts.SplashActivity}"

**Review by daywalk3r666 — CHANGES_REQUESTED** [review]

**Comment by daywalk3r666** [comment]
The two icons has the wrong activities

**Comment by Chefski** [comment]
@peterge1998 try applying the suggested changes and see if they're still not working.

**Review by Chefski — APPROVED** [review]

**Pull request merged**